File: IOS/SAIOT/SAIOT/DataModel/LivingSensor.swift
//
//  LivingSensor.swift
//  SAIOT
//
//  Created by 양창엽 on 2018. 4. 2..
//  Copyright © 2018년 Yang-Chang-Yeop. All rights reserved.
//

import RealmSwift

class LivingSensor: Object {
    
    // MARK: - Date Value
    @objc dynamic var collectDate: Date = Date()
    @objc dynamic var occurSparkDate: Date = Date()
    
    // MARK: - Integer Value
    @objc dynamic var temputure: Int = 0
    @objc dynamic var humidity: Int = 0
    @objc dynamic var noise: Int = 0
    @objc dynamic var gas: Int = 0
}

File: Android/SAIOT/app/src/main/java/androidtown/org/saiot/LivingSensor.java
package androidtown.org.saiot;

import java.util.Date;
import io.realm.RealmObject;

/**
 * Created by Ei Seok on 2018-04-24.
 */

public class LivingSensor extends RealmObject {

    //Date Value
    private Date collectDate = new Date();
    private Date occurSparkDate = new Date();

    //Integer Value
    private int temputure;
    private int humidity;
    private int noize;
    private int gas;

    //public String getName() { return name; }
    //public void   setName(String name) { this.name = name; }
    public int    getTemputure() { return temputure; }
    public void   setTemputure(int temputure) { this.temputure = temputure; }
    public int    getHumidity() { return humidity; }
    public void   setHumidity(int humidity) { this.humidity = humidity; }

    public int getNoize() {return noize;}
    public void setNoize(int noize){
        this.noize=noize;
    }

    public int getGas() {return gas;}
    public void setGas(int gas){
        this.gas=gas;
    }


}
